jarviscli/plugins/pi.py

```diff
@@ -10,17 +10,17 @@
 
 @plugin("pi")
 def next_pi(jarvis, s):
-    jarvis.say("Pi equals 3.14...", Fore.GREEN)
-    # today = datetime.date.today()
-
-    # print(today)
-
-    pi_file = open(os.path.join(FILE_PATH, '../data/pi.txt'), 'r')
-    pi_number = pi_file.read()
-    index = 4
-    while True:
-        user_input = jarvis.input("Enter \'n\' to print next {} digits : ".format(str(NUM_NEXT)))
-        if (user_input != 'n'):
-            break
-        jarvis.say(pi_number[index: index + NUM_NEXT], Fore.GREEN)
-        index += NUM_NEXT
+    """
+    Give the number of digit for pi
+    """
+    if s == "":
+        s = jarvis.input("Number of digit after dot? ")
```

Comment:
Again here please change 'digit' to digits (plural). Also, need a 'the' here so should be 'Number of digits after **the** decimal point'.

Reply:
I corrected it thank you for pointing out :)